### DISCUSSION TITLE
duplicate files in playlist order

### DISCUSSION THREAD

**Discussion by Marcioghiraldelli** [AzuraCast/AzuraCast#6365] [General]
Hello, I have noticed that Azuracast does not allow the panel to duplicate the same file without you creating several of the same file with different names.

Example
start
music
vignette
music
vignette
music
end

in case the vignette is the same audio, I would have to send the same audio with a different name to accept it in the playlist that way.

Now if I download the playlist in m3u8 and edit the file in notepad I can do any manual sequence, save and import it in the playlist and it works!!

So I decided to play with the azuracast api and when inserting songs through the api into a playlist, if I try to insert the same audio twice to avoid creating the same audio with a different name, I can't either.

Does anyone have any idea of any api of azuracast that allows you to do this?

**Comment by BusterNeece** [comment]
This should be allowed with sequential playlists, but not with other playlist types, as adding the same song multiple times wouldn't make any sense in that context.

**Reply by Marcioghiraldelli** [reply]
thanks for answering
this is precisely my question, the api does not allow me to insert the same audio several times and organize it without "physically" duplicating the audio, now if I do it through the m3u8 file it accepts it.
How could I send a duplicated audio via api? because if I send two he only registers one.
I didn't see how to do it with Azuracast, in fact I ran everything and I didn't find a function to do that. so i decided to go for api.

**Reply by Marcioghiraldelli** [reply]
I don't know if I'm making myself clear enough.
for example, I use the sequential list and I want to organize that the same file is repeated within the playlist.
currently I can't do that without being through m3u8 which becomes too much work.
so I saw that azura doesn't do that, so I resorted to the api where I create the list in php , convert it to json and send it via api. But if I have two or more audios with the same name api do azuracast just register one, it does not duplicate.
Can you give me a tip on how to solve this?
